src/plugins/formvalid/formvalid-en.hbs

```diff
@@ -351,6 +351,23 @@
 				</details>
 			</fieldset>
 
+			<div class="form-group">
+				<div class="checkbox checkbox-bold required">
+					<label for="agree1"><input id="agree1" name="agree1" type="checkbox" required> <span class="field-name">I agree</span> <strong class="required">(required)</strong></label>
+				</div>
+			</div>
+			<details class="mrgn-bttm-md">
+				<summary>View code</summary>
+				<pre><code>&lt;div class="wb-frmvld"&gt;
+	&lt;form action="#" method="get" id="validation-example"&gt;
+		...
+			&lt;div class="form-group"&gt;
+				&lt;div class="checkbox checkbox-bold required"&gt;
```

Comment:
@EricDunsworth Please try to come up with a name based on the function and not the effect, such as: `checkbox-standalone`

Reply:
Just forced-pushed an update that renamed it from ``checkbox-bold`` to  ``checkbox-standalone``. Meant to do it back in December but ran out of time lol. Still need to get to documentation at some point.